org/cloudfoundry.yml

```diff
@@ -3678,6 +3679,7 @@ orgs:
         - ccjaimes
         - gururajsh
         - PeteLevineA
+        - noxsingvm
```

Comment:
Note that this team will be going away with https://github.com/cloudfoundry/community/pull/387